cs/database-core.texy

```diff
@@ -28,7 +28,7 @@ Nicméně šikovnější způsob nabízí [aplikační konfigurace |configuring#
 Poté objekt spojení [získáme jako službu z DI kontejneru |di-usage], např.:
 
 /--php
-class Model
+class PostModel
```

Comment:
Proč Post?

Reply:
Prejmenoval jsem to na FooModel. Obecne pak lidi maji vsude jenom Model.php tridy a jsou z toho akorat zmateni.